pkg/portutil/port_allocate.go

```diff
@@ -0,0 +1,26 @@
+//go:build !linux
```

Comment:
Rename `port_allocate.go` to `port_allocate_linux.go`, and you can drop this line